src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetWidget.java

```diff
@@ -854,13 +861,20 @@ private void manageToolbarSizes()
       knitDocumentButton_.setText(width < 450 ? "" : knitCommandText_);
       
       if (editor_.getFileType().isRd() || editor_.getFileType().isJS() || 
-          editor_.getFileType().isSql() ||editor_.getFileType().canPreviewFromR())
```

Comment:
Why are we removing the `canPreviewFromR()` check here? (seems like it's orthogonal to this feature)